ext/opentelemetry-exporter-cloud-trace/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -54,15 +54,23 @@
 from opentelemetry.sdk.trace import Event
 from opentelemetry.sdk.trace.export import Span, SpanExporter, SpanExportResult
 from opentelemetry.sdk.util import BoundedDict
+from opentelemetry.sdk.version import __version__ as core_version
 from opentelemetry.util import types
 
+# pylint: disable=import-error
```

Comment:
`pylint ext/opentelemetry-exporter-cloud-trace/src/opentelemetry/exporter/cloud_trace/__init__.py` worksforme without this. Do you need it?

Reply:
Strange wasn't working for me. I suppose we'll see what travis thinks.

Reply:
Travis is complaining. When running linting stuff locally
`tox -e lint` produces different results then running `black`, `isort`, `pylint` sequentially. Tox seems to complain about relative imports, but not pylint by itself.